### ISSUE TITLE
Casting via Sessions API

### ISSUE THREAD

**Issue by GioPan04** [jarnedemeulemeester/findroid#594]
Jellyfin has a really useful feature that allows you to cast an item to a device using the [sessions api](https://api.jellyfin.org/#tag/Session/operation/Play).
This allows to remotely control every Jellyfin client, like the Web UI or for example the [official mpv plugin](https://github.com/jellyfin/jellyfin-mpv-shim).
It would be really nice if this client could interact with this API to cast movies or tv series to another device.

**Comment by Jcuhfehl** [comment]
I agree that this would be a great feature. I tried looking into it once, but I couldn't figure out the api, so if someone knows where that is documented I could try to implement it

**Comment by GioPan04** [comment]
I think their doc is good enough. I used it to create a simple PoC in Flutter. Here is the link to their doc: https://api.jellyfin.org/#tag/Session/operation/Play

**Comment by wrenix** [comment]
For me the only missing feature ;)
